packages/web-app-files/src/components/AppBar.vue

```diff
@@ -306,6 +300,19 @@ export default {
       this.createModal(modal)
     },
 
+    newButtonDescription(isToolTip) {
```

Comment:
While this is totally fine technically, it is hard to read in the location where it's called. `this.newButtonDescription(true)` or `this.newButtonDescription(false)` doesn't give a hint what effect the boolean has on the output. I'd prefer a solution without the boolean, as pointed out in the two comments above.